qbft/proposal.go

```diff
@@ -77,7 +77,7 @@ func isValidProposal(
 		return errors.Wrap(err, "could not hash input data")
 	}
 	if !bytes.Equal(signedProposal.Message.Root[:], r[:]) {
-		return errors.Wrap(err, "H(data) != root")
+		return errors.New("H(data) != root")
```

Comment:
you can also do a global error var like this, if you do it please choose a better variable name

`ErrHashDataIsNotRoot = errors.New("H(data) != root")` and then
```suggestion
		return ErrHashDataIsNotRoot
```

Reply:
@aaron-blox I agree that it would reduce code repetitions but I didn't find any global errors in spec, so it would be the first one, which, I think, would make the error handling style a bit inconsistent throughout the project. I think, if we do that, we need to do that in a separate PR and search for other errors that are repeated.

@alonmuroch Your thoughts?